src/main/java/com/jmb/springfactory/service/task/TaskService.java

```diff
@@ -0,0 +1,24 @@
+package com.jmb.springfactory.service.task;
+
+import java.util.List;
+import java.util.Optional;
+
+import com.jmb.springfactory.exceptions.NotFoundException;
+import com.jmb.springfactory.exceptions.PersistenceLayerException;
+import com.jmb.springfactory.exceptions.ServiceLayerException;
+import com.jmb.springfactory.model.bo.QueryTaskObject;
+import com.jmb.springfactory.model.dto.CommentDto;
+import com.jmb.springfactory.model.dto.TaskDto;
+import com.jmb.springfactory.model.dto.WorkLogDto;
+import com.jmb.springfactory.service.GenericService;
+
+public interface TaskService extends GenericService<TaskDto, Integer> {
+
+  public TaskDto save(Integer orderId, TaskDto taskDto) throws ServiceLayerException, NotFoundException, 
+    PersistenceLayerException;
+  
+  public List<TaskDto> findAll(QueryTaskObject queryParams);
+  public void addComment(Integer idTask, CommentDto comment) throws NotFoundException, ServiceLayerException;
+  public void addWorkLog(Integer idTask, WorkLogDto workLog) throws NotFoundException, ServiceLayerException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)